examples/seismic/acoustic/wavesolver.py

```diff
@@ -111,14 +125,22 @@ def adjoint(self, rec, srca=None, v=None, m=None, **kwargs):
         Adjoint modelling function that creates the necessary
         data objects for running an adjoint modelling operator.
 
-        :param rec: Symbol with stored receiver data. Please note that
-                    these act as the source term in the adjoint run.
-        :param srca: Symbol to store the resulting data for the
-                     interpolated at the original source location.
-        :param v: (Optional) Symbol to store the computed wavefield
-        :param m: (Optional) Symbol for the time-constant square slowness
-
-        :returns: Adjoint source, wavefield and performance summary
+        Parameters
+        ----------
+        rec :
```

Comment:
reminder.  @mloubout , could you provide the exact data types to @georgebisbas

Reply:
SparseTimeFunction or Array

Reply:
rec : SparseTimeFunction or array-like